javascript/javascript1/week3/whenwillwe-1.js

```diff
@@ -0,0 +1,15 @@
+const travelInformation = {
+    speed: 50, 
+    destinationDistance: 432, 
+};
+
+function getArrival() {
+    const totalTimeInHours = travelInformation.destinationDistance / travelInformation.speed;
+    const hours = Math.floor(totalTimeInHours);
+    const minutes = Math.floor((totalTimeInHours - hours) * 60);
+    const seconds = Math.round(((totalTimeInHours - hours) * 60 - minutes) * 60);
+
+    return `${hours} hours, ${minutes} minutes, and ${seconds} seconds`;
```

Comment:
This is an excellent solution, but I do want to be a bit annoying and note that if it takes less than an hour, then I would not want to see `0 hours`. Furthermore, when there is only 1 of a time unit, then grammar is wrong (e.g., `1 hours`). 😄

How would you handle this? 🙂